app/components/SessionDepoiments/index.tsx

```diff
@@ -0,0 +1,94 @@
+'use client'
+import React, { useEffect, useState } from 'react'
+import style from './style.module.css'
+import './swiper.css'
+import { Navigation, Pagination, Scrollbar, A11y } from 'swiper/modules'
+import { Swiper, SwiperSlide } from 'swiper/react'
+
+// Import Swiper styles
+import 'swiper/css'
+import 'swiper/css/navigation'
+import 'swiper/css/pagination'
+import 'swiper/css/scrollbar'
+import { Card } from './Card'
+import { Tables } from '@/app/protocols'
+import { testimonialsContentType, testimonialsType } from './types'
+
+interface DepoimentsProps {
+  values: Tables<testimonialsContentType>[]
+}
+
+export const Depoiments: React.FC<DepoimentsProps> = ({ values }) => {
+  const testimonialsContent = values[0].fields
+  const [slidePerView, setSlidePerView] = useState(4)
+  const [showNavigation, setShowNavigation] = useState(true)
+  const [space, setSpace] = useState(80)
+  const handleResize = () => {
+    if (window.innerWidth < 768) {
+      setSlidePerView(1)
+      setShowNavigation(false)
+      setSpace(30)
+    }
+
+    if (window.innerWidth >= 768 && window.innerWidth < 1280) {
+      setSlidePerView(2)
+      setShowNavigation(true)
+      setSpace(40)
+    }
+    if (window.innerWidth >= 1280) {
+      setSlidePerView(3)
+      setShowNavigation(true)
+      setSpace(40)
+    }
+  }
+
+  useEffect(() => {
+    // Chama a função de redimensionamento ao carregar a página
+    handleResize()
+
+    // Adiciona o event listener para redimensionamento
+    window.addEventListener('resize', handleResize)
+
+    // Retorna uma função de limpeza para remover o event listener quando o componente é desmontado
+    return () => {
+      window.removeEventListener('resize', handleResize)
+    }
+  }, [])
+  return (
+    <section className={style.depoiments}>
+      <h1 className={`title ${style.title}`}>{testimonialsContent.title}</h1>
+      <h2 className={`title--extra-small`}>
+        {testimonialsContent.description}
+      </h2>
```

Comment:
Diria que aqui temos h2 e h3. O h1 da página é o conteúdo principal dela, todo o resto está teoricamente relacionado a ele. O FAQ é nossa entrega principal, é a finalidade principal da página, mas eu entendo que o h1 seria a primeira seção.

To me baseando em conteudos como esse daqui:
https://www.a11yproject.com/posts/how-to-accessible-heading-structure/

"Use one unique h1 per page that describes what that page is about. That h1 preferably starts just above the main content."

Reply:
Perfeito, tags alteradas.